packages/web/src/utils/passwordListLazyLoader.ts

```diff
@@ -0,0 +1,3 @@
+import commonPasswordList from 'fxa-common-password-list'
```

Comment:
I'm not entirely sure you need this extra file for the lazy loading to work

Reply:
you're probably right, but it seemed intuitive to split at the time